packages/client/src/tx/signed-extensions/mortal-enc.ts

```diff
@@ -9,12 +9,24 @@ function trailingZeroes(n: number) {
   return i
 }
 
+function nextPower(n: number) {
+  if ((n & (n - 1)) === 0) return n
+  let count = 0
+  while (n > 0) {
+    n >>= 1
+    count++
+  }
+  return 1 << count
+}
```

Comment:
```suggestion
function nextPower(n) {
  const log = Math.log2(n);
  return 1 << Math.ceil(log)
}
```

This should work just as fine and it's a bit shorter

Reply:
I don't think we throw on any other signed extension, but both options sound good to me.